tests/unit/services/gmail/test_delete.py

```
@@ -0,0 +1,533 @@
+        mock_service.users.return_value.messages.return_value.list.return_value = mock_list
+
+        scan_senders_for_delete(limit=100)
+
+        status = get_delete_scan_status()
+        assert status["message"] == "No emails found"
+        assert status["done"] is True
+        assert status["error"] is None
+
+    @patch("app.services.gmail.delete.get_gmail_service")
+    @patch("app.services.gmail.delete.time.sleep")
+    def test_successful_scan(self, mock_sleep, mock_get_service):
```

Ruff (0.14.10)

Unused method argument: `mock_sleep`

(ARG002)